### ISSUE TITLE
Introduce yourself...

### ISSUE THREAD

**Issue by susanBuck** [susanBuck/e2-fall21#8]
Hi everyone - 

I thought I'd kick off a post where everyone can reply with an introduction to themselves so we can get to know each other. 

**Tell us what brought you to this course and any other details you care to share.**

A little bit about myself - I'm a web programmer and educator, living in Arlington, MA.

In addition to this course, I also teach [DGMD E-28](https://hesweb.dev/e28) and [CSCI E-15](https://hesweb.dev/e15). I've also worked with students completing their capstones in the DGMD program.

In the past, I've taught at Wellesley, UPenn, and UMiami. When working in industry I was the head developer for an online e-commerce store. 

More about my background here: [susanbuck.net](https://susanbuck.net)

All in all, I love creating things with code and teaching others how to do the same.

Looking forward to getting to know you all!

**Comment by AbdellGautier** [comment]
Hi Professor Buck and fellow classmates,

My name is Abdell Gautier. I was born and raised in Puerto Rico and moved to Miami, Florida, in 2013. Currently, I work as a manager of application development.

I am pursuing an ALB in the field of computer science and would love to pursue the ALM in the field of software engineering. I have never programmed in PHP, so I am excited to learn about the language.

I am happy to connect via LinkedIn:
[https://www.linkedin.com/in/abdell-gautier-84aa8131/](https://www.linkedin.com/in/abdell-gautier-84aa8131/)

I am looking forward to the class and to meeting our classmates!

**Comment by owen-mcg** [comment]
Hello all! 

Contrary to my GitHub display name, my name is Frank. I'm pursuing a certificate through HES, and I'm hoping to switch careers into tech. I'm excited (and a little nervous) to be back in school; and I'm excited to learn with, and from, you all.

**Comment by mike32jacobs** [comment]
Hi,
My name is Mike Jacobs. I am a high school math teacher. I live and work in New Hampshire. I am also pursuing a certificate from the HES. I would like to start a computer programming curriculum at the school where I teach.

**Comment by harrison2705** [comment]
Hello all, 
My name is Huy Nguyen and I am from Melbourne, Australia. This is my final course before taking the capstone project in Digital Media. I'm looking forward to working with all of you this semester. 
Cheers

**Comment by abarry25** [comment]
Hi all,

My name is **Alexa Barry**. I work over at the Harvard Innovation Labs in Alumni Engagement. I took David Heitmeyer's Fundamentals of Web Development this summer and wanted to continue to expand my knowledge and understanding of web development. I am excited to work with you all this semester!

**Comment by susanBuck** [comment]
@mike32jacobs Kudos for working to bring comp. sci to school - that will really help give your students a leg up.

@abarry25 Heitmeyer's E-12 is a great precursor to this course. 👍

**Comment by lyonsco** [comment]
Hi All,

My name is Courtney Lyons, and I work in stewardship communications at Harvard Business School and am working toward an ALM in Digital Media Design. In the spring, I took Introduction to Creative Exploration on the Web, and I am hoping to build on the programming skills I learned in that course. Looking forward to learning with you all this semester!

**Comment by wtang12340** [comment]
Hello,

My name is William Tang, a mobile app developer in Boston and I'm taking this course as an elective towards my Information Management Systems ALM. I'm fairly interested in PHP and web programming so I chose this course and hope to learn a lot with everyone.

**Comment by mgarimelHES** [comment]
Hello **Friends!**,
 
 My name is *Murthy Garimella* and based out of Colorado and am working mostly in the leading public provider helping customers and partners in their journey towards hybrid/ multi cloud migration. 

 I am working on my ALM and I am bit new to PHP and looking forward to learn and help each other as a part of this journey.

**Comment by ngal2021** [comment]
Hi! My name is Nicola Gallagher, and I work for the Department of Environmental Health in the Harvard. T.H. Chan School of Public Health. 

My background is in public relations, health communications, digital marketing, and international development. Throughout my career, I've managed several websites and have utilized a variety of content management systems. 

I am taking this class because I want to improve my skillset and learn PHP to create/customize websites/web applications. Looking forward to growing my skills and advancing from knowing how to manage content for static pages to having the ability to create interactive, dynamic digital spaces!

**Comment by gotgits** [comment]
Hello all,
My name is Rose. I am *interested* in everything on the syllabus and the opportunity to expand my frontend experiences with server-side environments and syntax.

I am looking forward to being a part of this forum and everyone's input.

**Comment by ant513g** [comment]
Hi all,

My name is Ana and I am a freelance designer living in NYC. I am also taking this class for my ALM Digital Media Design.

I create websites where I have used the basics of PHP--enough to get by. I am interested in gaining a deeper understanding of PHP's capabilities as well as back-end development to create the best websites for my clients.

**Comment by gabichuela85** [comment]
Hi All, 

My name is Gaby and I work at Harvard's Center of Mathematical Sciences and Applications. I'm currently working in finance, but have taken a number of COMPSCI courses at the **Extension school** over the past year. I am hoping to complete a graduate certificate soon. 

I took the *Wordpress* course at the **Extension** last fall, and while we touched on PHP, we didn't do a deep dive into it. I'm looking forward to learning more about PHP and to using GitHub more for my projects.
